.circleci/config.yml

```diff
@@ -107,28 +117,34 @@ jobs:
       - run: yarn build:webpack
 
   build-e2e:
+    working_directory: ~/vets-website
     docker:
       - image: cypress/base:10.15.3
     steps:
       - checkout
       - restore_cache:
           keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
+            - yarn-packages-cypress-{{ checksum "yarn.lock" }}
```

Comment:
I made a separate cache key for caching both Yarn and Cypress. This prevents Cypress cache from being stored in the general `yarn-packages-{{ checksum "yarn.lock" }}` cache. The Cypress binary that's cached adds about 100MB to the cache, and isn't needed for the non e2e jobs.

Reply:
Ooh, good catch.

Reply:
For nomenclature, could we be general and go with `yarn-packages-e2e`?

Reply:
Good idea, I'll rename it.